puma/apps/android/telegram/telegram.py

```diff
@@ -230,10 +231,11 @@ def end_call(self):
         """
         Ends the current call. Assumes the call screen is open.
```

Comment:
it is not possible to access what screen is open right now right ?

Reply:
Yes we can, and we check that below. The "assumes call screen is open" is mentioned here because we also have functions that handle going to the correct screen, for instance when sending a message. For calling this is not feasible, because you would need to also start a call in order to be in the call screen, which is a separate action.